src/binder/utils.cpp

```diff
@@ -300,17 +269,21 @@ namespace BINDER_SPACE
         BINDER_LOG_STRING(W("path A"), pathA);
         BINDER_LOG_STRING(W("path B"), pathB);
 
-        WCHAR tempResultPath[MAX_LONGPATH];
-        if (PathCombineW(tempResultPath, pathA, pathB))
-        {
-            combinedPath.Set(tempResultPath);
-            BINDER_LOG_STRING(W("combined path"), tempResultPath);
-        }
-        else
+        PathString tempResultPath;
```

Comment:
We could combine it directly into the combinedPath instead of creating the tempResultPath.